auth.go

```diff
@@ -210,12 +210,11 @@ type Central struct {
 	// on their operands.
 	Stats                  CentralStats
 	ldap                   LDAP
-	authenticator          Authenticator
 	userStore              UserStore
 	permitDB               PermitDB
 	sessionDB              SessionDB
 	roleGroupDB            RoleGroupDB
-	renameLock             sync.Mutex
+	mutex                  sync.Mutex
```

Comment:
The first rule of locks is that it needs to be *extremely* clear what the lock governs. Imagine a house covered in locks, but you don't know which lock opens which door. That's what it's like when you see a variable named "mutex". Unless it's painfully obvious, every mutex/lock need to have a comment next to it, explaining what it guards.